h2o-docs/src/product/automl.rst

```diff
@@ -85,7 +85,7 @@ Optional Miscellaneous Parameters
     - ``GLM``
     - ``DeepLearning``
     - ``GBM``
-    - ``DRF`` (includes both the Random Forest and Extremely-Randomized Trees models)
+    - ``DRF`` (This includes both the Random Forest and Extremely-Randomized Trees (XRT) models. Refer to the ``histogram_type`` parameter description in the `DRF chapter <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/drf.html#defining-a-drf-model>`__ and in the `Parameters Appendix <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/histogram_type.html>`__ for more information.)
```

Comment:
I think we should add a separate section in DRF for XRT (not just in the `histogram_type` description) and link to that instead here.